GHzDACs/ghz_fpga_server.py

```diff
@@ -195,6 +195,9 @@
 import itertools
 import struct
 import time
+import logging
+logging.basicConfig(level=logging.WARNING)
```

Comment:
I'm not sure best practices for python, but could we put this in the `__name__ == '__main__'` block instead of at the module top-level?